action.yml

```diff
@@ -56,6 +56,11 @@ inputs:
     default: "v"
     required: false
 
+  infile:
+    description: "Read the CHANGELOG from this file"
+    default: "CHANGELOG.md"
```

Comment:
Also wondering what happens here? Wil this now not cause the whole old changelog to be append to the new one? (if you generate more then only the latest release, 5 is currently the default)

Think it's better to not have a default here?

Reply:
OK updated